app/sequencescape_excel/sequencescape_excel/specialised_field/sanger_tube_id.rb

```diff
@@ -18,32 +18,33 @@ class SangerTubeId
       # rubocop:todo Metrics/PerceivedComplexity
```

Comment:
Not sure we still need two separate classes here now tubes/plates are both labware with receptacles. But not worth touching at the moment.